plugins/modules/elb_classic_lb_info.py

```diff
@@ -177,11 +406,30 @@ def describe_elb(connection, lb):
 
 @AWSRetry.jittered_backoff()
```

Comment:
@tremble @mandar242 @markuman  What do you think about moving some of these utilities to a module_utils folder in community.aws and calling it probably module_utils/elb.py or module_utils/elb_utils.py (I have a proposal to rename the actual elb_utils.py to elbv2_utils.py https://github.com/ansible-collections/amazon.aws/pull/2285)?  This will give a good start that can be further explored once the module and utility are migrated to amazon.aws and will ease the testing process, since we do not need cross-repo testing for the time being. I would like us to follow a similar pattern to what we have in amazon.aws.

Reply:
Personally I'd do the move after we've moved the code to amazon.aws, cleaning up both the elb_classic_lb and elb_classic_lb_info modules at the same time.

Reply:
@alinabuzachis , I agree with @tremble, once `elb_classic_lb_info.py` is moved to `amazon.aws` then module_utils for both `elb_classic_lb and elb_classic_lb_info` can be refactored in a better way in my opinion.